pkg/document/template.go

```diff
@@ -128,7 +128,7 @@ func getMaintainersTemplate() string {
 	maintainerBuilder.WriteString("| Name | Email | Url |\n")
 	maintainerBuilder.WriteString("| ---- | ------ | --- |\n")
 	maintainerBuilder.WriteString("  {{- range .Maintainers }}")
-	maintainerBuilder.WriteString("\n| {{ .Name }} | {{ .Email }} | {{ .Url }} |")
+	maintainerBuilder.WriteString("\n| {{ .Name }} | {{ .Email }} | <{{ .Url }}> |")
```

Comment:
Please could you decorate the `{{ .Emile }}` the same way.

Reply:
Done, sorry for the delay!